cypress/local-only/tests/integration/statusReportOrder/edit-status-report-order.cy.ts

```diff
@@ -165,6 +165,10 @@ describe('edit status report order', () => {
       cy.contains('a', messages.testStatusReportOrderSigned.name).click();
       cy.get('[data-testid="edit-signed-document-button"]').click();
       cy.get('[data-testid="modal-button-confirm"]').click();
+      cy.get('#status-report-due-date-picker').clear();
+      cy.get('#status-report-due-date-picker').type(
+        formatNow(FORMATS.MMDDYYYY),
+      );
```

Comment:
This test was secretly relying on other tests to run first and update the date. These lines fix that.